### PR TITLE
Fix: Button issues

### PR DESCRIPTION
**Pull request by iceicery**
This PR fix: 
1. Wrong pressed state for the Request demo button 
2. Increase the left and right paddings for the Drop us a line button
3. Wrong pressed state for the Contact us button (design oversight, see the project Library)

All based on Elements of Figma https://www.figma.com/file/m3nYO4Yg9kLTGWvzoHrPKY/Iterative.ai?node-id=11%3A109 (Not the Master of Figma)

### CODE REVIEW

File: src/components/Button/Button.css [hunk 1]
```diff
@@ -31,6 +31,7 @@
 .button:focus {
```
Comment (julieg18) [inline src/components/Button/Button.css:31]: Actually thinking about it, we may need to update our button pseudo selectors...giving the hover styles to the focus and the pressed styles to the active... Let's discuss this later! 😄

File: src/components/Button/Button.css [hunk 2]
```diff
@@ -31,6 +31,7 @@
...
   box-shadow: inset 0px 2px 1px rgba(46, 49, 55, 0.15),
     inset 0px 0px 4px rgba(20, 20, 55, 0.3);
+  outline: none;
```
Comment (julieg18) [inline src/components/Button/Button.css:34]: Why is this outline taken off? It's rather difficult to see when a button is being focused on when it's just a box shadow.

File: src/components/Button/Button.css [hunk 3]
```diff
@@ -47,11 +48,7 @@
 }
 
 .button:hover.button_color_blue-gradient {
-  filter: brightness(1.1);
-}
-
-.button:focus.button_color_blue-gradient {
-  filter: brightness(1);
+  background: linear-gradient(270deg, #00dbd8 0%, #b133ff 100%);
```
Comment (julieg18) [inline src/components/Button/Button.css:51]: The hover style blocks out the focus styles.

File: src/components/Button/Button.css [hunk 4]
```diff
@@ -64,10 +61,9 @@
 }
 
 .button:hover.button_color_orange-gradient {
-  filter: brightness(1.1);
+  background: linear-gradient(270deg, #ffa82e 0%, #c747eb 100%);
```
Comment (julieg18) [inline src/components/Button/Button.css:64]: The orange gradient hover style still works when a button is disabled.

File: src/components/Button/Button.css [hunk 5]
```diff
@@ -126,9 +122,9 @@
   }
 
   .button_size_medium {
-    padding: 16px 24px;
-    font-size: 20px;
-    line-height: 20px;
+    padding: 8px 16px;
```
Comment (julieg18) [inline src/components/Button/Button.css:125]: Nice job on fixing the sizing!

**Review by julieg18 — CHANGES_REQUESTED** [review]
I noticed that the transition doesn't work on changing linear gradients. We could implement a way for it to work with this: https://keithjgrant.com/posts/2017/07/transitioning-gradients/ . We could save this for a later time, though 😃

File: src/components/Button/Button.css (hunk 2 above)
Comment (iceicery) [reply src/components/Button/Button.css:34]: I know. But I noticed that the reviewers asked us to take off many outline( Logo, Product menu...). They asked us to change the pressed state but it has nothing else to change.... That is why I took off outline. What do you think?

File: src/components/Button/Button.css [hunk 6]
```diff
@@ -42,34 +44,58 @@
...
+  bottom: 0;
+  left: 0;
+  z-index: -1;
+  transition: opacity 0.5s linear;
+  opacity: 0;
+  background: linear-gradient(270deg, #ffa82e 0%, #c747eb 100%);
+  border-radius: 100px;
+}
+
+.button:hover:not(:focus):not(:disabled).button_color_orange-gradient::before {
```
Comment (julieg18) [inline src/components/Button/Button.css:93]: Great idea using `:not`!

**Review by julieg18 — APPROVED** [review]
Great job! It looks brilliant1

**Pull request merged**